core/src/main/java/org/jdbi/v3/core/result/ResultIterable.java

```diff
@@ -76,6 +93,8 @@
     @Override
     ResultIterator<T> iterator();
 
+    StatementContext getContext();
```

Comment:
Nope, was vestigial from an intermediate state.  I'll take it out.  It is weird that you can get the context at points before and after but not here though.